.github/workflows/workflow.yml

```diff
@@ -26,6 +26,8 @@ jobs:
 
       - name: Mise
         uses: jdx/mise-action@v2
+        with:
```

Comment:
I also want to demonstrate this issue: https://github.com/jdx/mise/discussions/5934

However, the version of `mise` being pulled into the action is too old so I pushed it to the latest in the action (this should be removed before merging).